src/modules/imageresizer/ui/Models/ResizeOperation.cs

```diff
@@ -97,6 +105,12 @@ public void Execute()
 
                     var frame = CreateBitmapFrame(transformedBitmap, metadata);
```

Comment:
Why double create. You could declare frame then set via an if else

Reply:
That's a great point.  Thanks for your help!  I can restructure my code to avoid the GetValidMetadata call and related code. 

Instead of setting a boolean and checking it later in the code, would this be a better approach?  I think this would be a bit more performant: 
`                    if (transformedBitmap == originalFrame)
                    {
                        var frame = CreateBitmapFrame(originalFrame, originalMetadata);
                        encoder.Frames.Add(frame);
                        continue;
                    }`

Reply:
Actually .... I would do this looking at the diff more.  A lot of the meta data work isn't needed if the frame is skipped.

```
                    BitmapFrame frame;
                    // if the frame was not modified, we should not replace the metadata
                    if (transformedBitmap == originalFrame)
                    {
                        frame = CreateBitmapFrame(originalFrame, originalMetadata);
                    }
                    else
                    {
                        BitmapMetadata originalMetadata = (BitmapMetadata)originalFrame.Metadata;

#if DEBUG
                        Debug.WriteLine($"### Processing metadata of file {_file}");
                        originalMetadata.PrintsAllMetadataToDebugOutput();
#endif

                        var metadata = GetValidMetadata(originalMetadata, transformedBitmap, containerFormat);

                        if (_settings.RemoveMetadata && metadata != null)
                        {
                            // strip any metadata that doesn't affect rendering
                            var newMetadata = new BitmapMetadata(metadata.Format);

                            metadata.CopyMetadataPropertyTo(newMetadata, "System.Photo.Orientation");
                            metadata.CopyMetadataPropertyTo(newMetadata, "System.Image.ColorSpace");

                            metadata = newMetadata;
                        }

                        frame = CreateBitmapFrame(transformedBitmap, metadata);
                    }

                    encoder.Frames.Add(frame);
```